### PR TITLE
set default inline mapping charset to utf-8 and provide option to configure

### PR DESCRIPTION
**Pull request by jasonslyvia**
alse fixed some failing tests

### CODE REVIEW

**Comment by thlorenz** [comment]
Looks good, could you please remove all white space changes though?

**Comment by jasonslyvia** [comment]
sure, Sublime Text should take credits for that[?]

2015-01-06 20:04 GMT+08:00 Thorsten Lorenz <notifications@github.com>:

**Comment by jasonslyvia** [comment]
done

File: test/inline-source-map.js [hunk 1]
```diff
@@ -304,11 +305,38 @@ test('given mappings, with one having no original', function (t) {
...
+    );
+
+    t.end();
+  });
+
+  t.test('default charset should be utf-8', function(t){
+    var gen = generator()
+              .addGeneratedMappings('foo.js', foo);
+
+    t.deepEqual(
```
Comment (thlorenz) [inline test/inline-source-map.js]: Use `t.equal` when comparing strings

File: test/source-content.js [hunk 2]
```diff
@@ -40,19 +40,20 @@ test('generated mappings', function (t) {
...
             "function foo() {\n  var hello = 'hello';\n  var world = 'world';\n  console.log('%s %s', hello, world);\n}"
-          ]
+          ],
+          "sourceRoot": ""
         }
       , 'includes source content'
     )
 
     t.deepEqual(
-        decode(gen.base64Encode())
```
Comment (thlorenz) [inline test/source-content.js]: Use t.equal when comparing strings

**Comment by jasonslyvia** [comment]
Oops, miss that, will fix soon

**Comment by jasonslyvia** [comment]
not really familiar with tap, thanks for your review

**Comment by shinnn** [comment]
When will this be merged?

**Comment by thlorenz** [comment]
Thanks went out as 0.4

**Pull request closed without merging**